File: src/main/java/com/brasil/transparente/api/repository/UnidadeGestoraRepository.java
package com.brasil.transparente.api.repository;

import com.brasil.transparente.api.entity.UnidadeGestora;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface UnidadeGestoraRepository extends JpaRepository<UnidadeGestora, String> {

    List<UnidadeGestora> findByOrgaoOrgaoId(Long orgaoId);

}

File: src/main/kotlin/com/brasil/transparente/api/repository/UnidadeGestoraRepository.kt
package com.brasil.transparente.api.repository

import com.brasil.transparente.api.entity.UnidadeGestora
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface UnidadeGestoraRepository : JpaRepository<UnidadeGestora, String> {

    fun findByOrgaoOrgaoId(orgaoId : Long?) : List<UnidadeGestora>

}